packages/clerk-js/src/core/auth/SessionCookieService.ts

```diff
@@ -52,33 +87,24 @@ export class SessionCookieService {
   }
 
   private async refreshSessionToken(): Promise<void> {
-    if (!inBrowser()) {
-      return;
-    }
-
     if (!this.clerk.session) {
       return;
     }
 
     try {
-      this.updateSessionCookie(await this.clerk.session?.getToken());
+      await this.clerk.session.getToken();
```

Comment:
ℹ️ Since the `this.clerk.session.getToken()` triggers the `events.TokenUpdate` event which updates the session cookie there is no need to invoke `this.updateSessionCookie`.